### PR TITLE
[fix] convert to timezone aware datetime for execution time in the api

### PR DESCRIPTION
**Pull request by kristenmills**
### Motivation for changes:
Since we store the the datetimes without timezones, when they are json encoded in the api, it is assumed that they are in UTC (which they aren't). This adds the system local timezone before json encoding,  so that it will properly offset the time

### CODE REVIEW

File: flexget/components/status/db.py [hunk 1]
```diff
@@ -48,11 +48,16 @@ def last_execution_time(cls):
             select([func.max(TaskExecution.start)])
             .where(TaskExecution.task_id == cls.id)
             .correlate(StatusTask.__table__)
-            .label('last_execution_time')
+            .label('last_execuftion_time')
```
Comment (cvium) [inline flexget/components/status/db.py]: Is this change intentional?

Comment (kristenmills) [reply flexget/components/status/db.py]: nope thanks

**Comment by metaMMA** [comment]
I just wanted to say thanks for taking this on. I think this will be a useful feature!

File: flexget/components/status/db.py [hunk 2]
```diff
@@ -52,7 +52,12 @@ def last_execution_time(cls):
         )
 
     def to_dict(self):
-        return {'id': self.id, 'name': self.name, 'last_execution_time': self.last_execution_time}
+        return {
+            'id': self.id,
+            'name': self.name,
+            'last_execution_time': self.last_execution_time
+            and self.last_execution_time.astimezone(),
```
Comment (cvium) [inline flexget/components/status/db.py]: Is this a new syntax though? `last_execution_time` should be a datetime? But won't the `and` make it a bool?

Comment (kristenmills) [reply flexget/components/status/db.py]: `and` does not coerce the value to a bool. `a and b` is equivalent to if `a` is truthy return `b` else return `a`.

Comment (cvium) [reply flexget/components/status/db.py]: I do think `self.last_execution_time.astimezone() if self.last_execution_time else None` is easier to read though.

**Comment by gazpachoking** [comment]
LGTM!
